### ISSUE TITLE
Video Portal: video with several tracks: error in the duration of the videos

### ISSUE THREAD

**Issue by agusave82** [campusdomar/PuMuKIT#89]
The steps are as follows:
1. go to the back-office
2. upload a video with several tracks
3. transcode to profile h_264

![video_with_several_tracks](https://cloud.githubusercontent.com/assets/17157560/13638552/0add0114-e60d-11e5-9889-282533aaea92.png)
1. publish the video
2. Go to video portal

![video_with_several_tracks_video_portal](https://cloud.githubusercontent.com/assets/17157560/13638563/1add34a8-e60d-11e5-8ea8-af78354a5c50.png)

Error:

In Video Portal the duration is the same,

**Comment by Alfro** [comment]
@rubenrua This happens after the changes due to the editor trimming videos

**Comment by Alfro** [comment]
**TL;DR:** Having tracks with different duration is neither supported nor recommended. We should leave it more clear on the UI.

The reason for having different tracks on a multimedia object is so a same multimedia object can have different video qualities, resources on different languages, an only audio version or even two sources that are visualized simultaneously (for lectures and such). Also, the recently added **soft-editing** capabilities do not have into account different track durations.

I believe we should probably **remove** that 'duration' value on the series list and add it once, either on top of the thumbnail or in its own separated line.

**Comment by Yurujai** [comment]
Solved in version 3.0.
